.brigade/brigade.ts

```diff
@@ -310,7 +337,8 @@ async function runSuite(event: Event): Promise<void> {
       buildWorkerJob(event),
       buildBrigadierJob(event),  
       buildCLIJob(event)
-    )
+    ),
+    testIntegrationJob(event)
```

Comment:
Only test this after all of the less resource-intensive tests have passed.

I might revisit this decision in the future. Since everything gets _re-built_ in this step, maybe we can skip some of the preceding build jobs, knowing that this job covers our bases.